pkg/engine/handlers/validation/validate_resource.go

```diff
@@ -185,7 +183,7 @@ func (v *validator) validateOldObject(ctx context.Context) (*engineapi.RuleRespo
 	oldResource := v.policyContext.OldResource()
 	emptyResource := unstructured.Unstructured{}
 
-	if ok := matchResource(oldResource, v.rule); !ok {
+	if ok := matchResource(oldResource, v.rule, v.policyContext.NamespaceLabels(), v.policyContext.Policy().GetNamespace(), kyvernov1.Create); !ok {
```

Comment:
Can we use `v.policyConext.Operation()` instead of `kyvernov1.Create`? Or is it always a CREATE operation?

Reply:
It is always `CREATE`, since its an old object we assume that during its validation